src/pages/Home/Home.css

```diff
@@ -1,13 +1,13 @@
 .landing {
-    height: 100vh;
-    background-color: #32936F;
+  height: 100vh;
+  background-color: #32936f;
 
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
 
-    font-size: 64px;
+  font-size: 64px;
```

Comment:
this looks like some formatting changes. remember to run `npm run format` to keep things like spacing and hex values (ie capital vs. lowercase) consistent.